main.nf

```diff
@@ -55,22 +89,77 @@ if (params.help) {
 /*
  * SET UP CONFIGURATION VARIABLES
  */
+if (params.input) { ch_input = file(params.input, checkIfExists: true) } else { exit 1, "Samplesheet file not specified!" }
+
+def ch_guppy_model = Channel.empty()
+def ch_guppy_config = Channel.empty()
+if (!params.skip_basecalling) {
+
+    // TODO pipeline: Add in a check to see if running offline
```

Comment:
Is that resolved / tackled? if not, you could generate an issue and try it out  / add something to the pipeline. I fear its getting lost otherwise ;-)

Reply:
Ill ask @pditommaso tomorrow 😎

Reply:
https://github.com/nf-core/nanoseq/pull/60/commits/f7d59001e3103d0fce0c186685ae64d713a152c1

Woooohoooooo.